src/model/person.ts

```diff
@@ -33,6 +34,69 @@ export class Person extends BackmatterEntity {
     }
   }
 
+  public toXml(affiliations: Affiliation[] = []): Element {
+    const xmlDoc = new DOMImplementation().createDocument(null, null);
+    const contrib = xmlDoc.createElement('contrib');
+    contrib.setAttribute('contrib-type', 'author');
+    contrib.setAttribute('id', this.id);
+    if (this.isCorrespondingAuthor) {
+      contrib.setAttribute('corresp', 'yes');
+    }
+  
+    const name = xmlDoc.createElement('name');
```

Comment:
Might be nothing to worry about, but there is also a [group author to consider](https://github.com/libero/editor/issues/63#issuecomment-538898308). Note the `collab` rather than name.

Reply:
Doesn't look like the client has anything set up to identify an author entry as being a collab group yet. I'd prefer to raise this as a feature issue for the future and leave it out of this issue for now.

Reply:
As it's already a user story issue I'll resolve this for now. We can address when we get round to breaking down issue libero/editor#63